components/file_upload/file_upload.jsx

```diff
@@ -385,9 +377,19 @@ export class FileUpload extends PureComponent {
         this.props.onFileUploadChange();
     }
 
-    registerDragEvents = (containerSelector, overlaySelector) => {
-        const self = this;
+    handleDropText = (e) => {
+        const targetType = e.target.nodeName.toLowerCase();
```

Comment:
nit: can we rename to targetNode or targetName? given there is an `e.target.type` I would expect targetType to contain that info.

Reply:
I feel like I had that exact thought at some point, so I can't remember why I didn't change that before.